plugins/modules/ovirt_disk.py

```diff
@@ -355,7 +355,8 @@ def create_transfer_connection(module, transfer, context, connect_timeout=10, re
         connection.connect()
     except OSError as e:
         # Typically ConnectionRefusedError or socket.gaierror.
-        module.warn("Cannot connect to %s, trying %s: %s", transfer.transfer_url, transfer.proxy_url, e)
+        module.warn("Cannot connect to {}, trying {}: {}"
+                    .format(transfer.transfer_url, transfer.proxy_url, e))
```

Comment:
Formatting messages for logs is bad idea. Formatting is expensive and should be done  only when the log message is actually logged. For warnings this is usually the case, but for debug message it is always not and having 2 ways to log in the same code is bad.

I hope this is not the way ansible scripts use logging. If it is I guess we want to keep this consistent with other logs here.

Reply:
This does is not for logging it just shows a user warning message. 
And around formatting, I don't mind to use it as it was before.